test/langtools/tools/doclint/html/TextNotAllowed.out

```diff
@@ -25,18 +25,33 @@ TextNotAllowed.java:14: error: text not allowed in <ul> element
 TextNotAllowed.java:14: error: text not allowed in <ul> element
  * <ul> abc <li> item </li> def <li> item </li> ghi </ul>
                                                ^
+TextNotAllowed.java:16: error: attribute not supported in HTML5: summary
+ * <table summary=description> abc </table>
+          ^
```

Comment:
The test is 'Text Not Allowed', so the test file should be valid except for text where it is not allowed.  Don't add spurious other errors. In this case, provide a `<caption>` to keep DocLint happy.

This applies throughout this test.

Reply:
Thanks.  I was lacking of such perspective.